cpa/tester/result.go

```diff
@@ -17,6 +17,25 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+var (
```

Comment:
Can we avoid using a global here? We prefer to avoid globals, even for testing, as they can complicate the understanding of the code flow.

Reply:
Oh sure! I changed the code, tell me what you think.